lma-main.yaml

```diff
@@ -314,6 +314,17 @@ Parameters:
       must pre-exist in your account,  match the Language Code selected above, and
       use the 'Narrow Band' base model.
 
+  ModelInitialization:
```

Comment:
I think this should this be 'ModelValidation' instead of 'ModelInitialization' ?